data/fixture/load.js

```diff
@@ -29,21 +29,27 @@ function loadFixtures(env, config, done) {
 
 	application(config, async function(error, app) {
 		if (error) {
+			return done(error);
```

Comment:
why does this one `return done()` when the one on line 52 just calls `done()` without the return? 🤔

Reply:
`done` doesn't itself return anything. So we do `return done()` to avoid having to write 
```
done();
return;
```

Reply:
The one on line 52 is the last line of the function (effectively) so the `return` is un-needed

Reply:
If `done()` doesn't return anything, isn't it clearer to write:

```
done();
return;
```
?

Reply:
Writing `return cb()` is a pretty standard practise in `node` apps IMO.